tasks/main.yml

```diff
@@ -112,18 +112,18 @@
 - name: Add mssql to the list of Tuned profiles
   when: '"mssql" not in __mssql_tuned_active_profiles.stdout'
   block:
-    - name: Add mssql to the list of Tuned profiles
-      command: >-
+    - name: Attempt to add mssql to the list of Tuned profiles
+      shell: >-
         tuned-adm profile {{ __mssql_tuned_active_profiles.stdout |
-        regex_replace( '^Current active profile: ', '' ) }} mssql
+        regex_replace( '^Current active profile: ', '' ) }} mssql || true
       register: __mssql_tuned_adm_profile
+      changed_when: __mssql_tuned_adm_profile.stderr | length == 0
```

Comment:
Try using a `failed_when: false` here instead of changing to `shell` with the ` || true`.

Here is the output of a failed `command` when using `failed_when: false`:
```
TASK [command] *****************************************************************************
task path: /home/rmeggins/ansible_sandbox/stat-test-1.yml:12
changed: [localhost] => {
    "changed": true,
    "cmd": [
        "ls",
        "nosuchfile"
    ],
    "delta": "0:00:01.003224",
    "end": "2021-06-30 13:52:32.131075",
    "failed_when_result": false,
    "rc": 2,
    "start": "2021-06-30 13:52:31.127851"
}

STDERR:

ls: cannot access 'nosuchfile': No such file or directory


MSG:

non-zero return code
```
still not great, but at least we don't have the `fatal` and `FAILED!`